### PR TITLE
Display errors

### PR DESCRIPTION
**Pull request by susiecoleman**
incorrect password
multiple errors

### CODE REVIEW

File: app/com/gu/identity/frontend/controllers/Application.scala [hunk 1]
```diff
@@ -12,9 +12,9 @@ class Application @Inject() extends Controller with Logging {
     Redirect(routes.Application.signIn())
   }
 
-  def signIn = Action {
+  def signIn = Action { req =>
     Cached{
-      Ok(renderSignIn)
+      Ok(renderSignIn(req.getQueryString("error").seq.toSeq, req.getQueryString("email").getOrElse("")))
```
Comment (jamespamplin) [inline app/com/gu/identity/frontend/controllers/Application.scala]: This was mentioned in #11, but we can add `email: Option[String]` and `error: Option[String]` to the `signIn` action, and add the parameter to the route, which might clean this up a bit. Not sure if you can do `error: Seq[String]`, but worth a try.

File: app/com/gu/identity/frontend/controllers/SigninAction.scala [hunk 2]
```diff
@@ -64,9 +64,14 @@ class SigninAction @Inject() (identityService: IdentityService) extends Controll
...
-    val query = errors.map(e => s"error=${e.message}").mkString("&")
-    SeeOther(routes.Application.signIn().url + s"?$query")
+  private def redirectToSigninPageWithErrorsAndEmail(errors: Seq[ServiceError], email: Option[String]) = {
+    val query = errors.map(e => s"error=${e.id}").mkString("&")
+    val emailAddress = email match {
+      case Some(email) => "&email=" + email
+      case None => ""
+    }
+    SeeOther(routes.Application.signIn().url + s"?$query" + emailAddress)
+      .withHeaders("Cache-Control" -> "no-cache")
```
Comment (jamespamplin) [inline app/com/gu/identity/frontend/controllers/SigninAction.scala]: @danmassie has created a `NoCache` wrapper on the `signIn` route, so this line is no longer necessary.

**Comment by jamespamplin** [comment]
A couple of small comments but looks good :+1: 

Also with client side changes, I like to put a screen grab on the PR for niceness.

File: app/com/gu/identity/frontend/controllers/SigninAction.scala (hunk 2 above)
Comment (susiecoleman) [inline app/com/gu/identity/frontend/controllers/SigninAction.scala]: fixed

File: app/com/gu/identity/frontend/controllers/Application.scala (hunk 1 above)
Comment (susiecoleman) [inline app/com/gu/identity/frontend/controllers/Application.scala]: fixed

**Comment by jamespamplin** [comment]
:+1: changes look good, thanks!

**Pull request merged**